src/els_db.erl

```diff
@@ -20,7 +20,7 @@
                 , els_dt_signatures
                 ]).
 -define(TIMEOUT, infinity).
--define(DB_SCHEMA_VSN, <<"2">>).
+-define(DB_SCHEMA_VSN, <<"3">>).
```

Comment:
Maybe we should use something in the line of semantic versioning for this. It makes more sense to me to bump a minor or a patch than bump the whole number. What do you think?

Reply:
The current check verifies exact equality and I don't see the advantage of having semver for the schema version. I am actually tempted by clearing the cache at every upgrade (so that we don't need to remember to bump this number), but that's not ideal either. My suggestion is to live with this for a while and see how things evolve.

Reply:
Perhaps we should use a date for this, so it gives some idea of when it changed?